### PR TITLE
Add genre select

### PR DESCRIPTION
**Pull request by NoamShuster**

### CODE REVIEW

File: src/spotifun.tsx [hunk 1]
```diff
@@ -16,11 +20,30 @@ export const Spotifun: React.FunctionComponent = (props => {
         }
     });
 
+    const onItemSelect = (selectedItem: React.MouseEvent) => {
```
Comment (NitsanAmit) [inline src/spotifun.tsx:23]: make this function of type (selectedItemId: string) =>void, and have the selectionBox component extract the id from the event

**Review by NitsanAmit — CHANGES_REQUESTED** [review]

File: src/genre-selection-box.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            backgroundImage={genre.imagePath}
+            isSelected={genre.selected}
+            onItemSelect={onItemSelect}
+        />
+    )
+}
+
+interface genreSelectionBoxProps {
+    genre: Genre,
+    onItemSelect: (event: any) => void
```
[the remark this answers is not in the document]
Comment (NoamShuster) [reply src/genre-selection-box.tsx]: fixed

File: src/icons/index.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import hipHop from './hip_hop.png';
+import electronic from './electronic.png';
+import sad from './sad.png';
+import happy from './happy.png';
+import dancey from './dancey.png';
+import angry from './angry.png';
+import relaxed from './relaxed.png';
+import workout from './workout.png';
+
+export {pop, rock, country, indie, metal,
```
[the remark this answers is not in the document]
Comment (NoamShuster) [reply src/icons/index.ts:16]: like you

File: src/spotifun.tsx (hunk 1 above)
Comment (NoamShuster) [reply src/spotifun.tsx]: fixed

**Review by NitsanAmit — APPROVED** [review]

**Pull request merged**